en/docs/manage/api-management.md

```diff
@@ -0,0 +1,54 @@
+# Manage
+
+Choreo's API Management capabilities allow you to manage your APIs through the Choreo Console. You can manage the lifecycle, consumers, and documents, secure your backends by applying rate--limiting policies through usage plans, define permissions using scopes, configure security, rate-limiting, and visibility settings, and define any properties. 
+
+# Overview
+
+The Overview of the Manage section gives you a bird's eye view of how you currently manage your API component. You can view the lifecycle status of your component, the number of consumers, the security scheme configured, and the usage plan applied to the API. The overview pane also lists the environment(s) of the API.
+
+# Lifecycle
+
+Choreo allows you to manage the lifecycle of the API through the API console. The states represent the stage where an API is in the process from creation to retirement. You can apply any of the six API lifecycles states: Created, Pre-released, published, blocked, deprecated, and retired while adhering to their respective lifecycle flows.
+
+When you publish an API, you can also publish a connector for that API to make it available on the Choreo Marketplace.
+
+# Consumers
+
+The consumers' pane shows us the applications that have subscribed to your API.  You can manage access to your API by blocking or unblocking these applications through the Choreo Console. The Consumers pane lists the applications, and for each application, it shows the usage tier, owner, and status: blocked or unblocked. 
```

Comment:
A pane is something a user can close or collapse while remaining on the page isn't it? How about using the term "**Consumers** section" instead?

Also, as a name of a UI element, the title **Consumers** should be in bold and formatted the same way as in the UI

Reply:
Not exactly collapsable or closable. It's just a section of the UI that is coupled to the rest of the regions.

Reply:
Made to be in bold.